File: java-advent-of-code/src/test/java/advent/twenty_sixteen/Day15TimingIsEverythingTest.java
package advent.twenty_sixteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day15TimingIsEverythingTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day15TimingIsEverything("/twenty_sixteen/Day15-TimingIsEverything-input.txt");

        assertEquals(148737, dp.getPart1Answer().intValue());
        assertEquals(2353212, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day15TimingIsEverything("/twenty_sixteen/Day15-TimingIsEverything-example#1.txt");

        assertEquals(5, dp.getPart1Answer().intValue());
        assertEquals(85, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_sixteen/Day15TimingIsEverythingTest.scala
package advent.twenty_sixteen

import org.scalatest.{FlatSpec, Matchers}

class Day15TimingIsEverythingTest  extends FlatSpec with Matchers {
  "2016 Day 15 - Input" should "calculate answers" in {
    val dp = new Day15TimingIsEverything("/twenty_sixteen/Day15-TimingIsEverything-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day15TimingIsEverything("/twenty_sixteen/Day15-TimingIsEverything-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
